.github/workflows/tests.yaml

```diff
@@ -23,15 +23,9 @@ jobs:
         - name: Install dependencies
           run: |
               python -m pip install --upgrade pip
-              pip install "numpy<=1.21" scikit-learn flake8 setuptools numba
-
-        - name: Install torch ${{ matrix.torch-version }} windows + linux
-          if: ${{matrix.os != 'macos-latest'}}
-          run: pip install torch==${{ matrix.torch-version }}+cpu -f https://download.pytorch.org/whl/torch_stable.html
-        - name: Install torch ${{ matrix.torch-version }} macos
-          if: ${{matrix.os == 'macos-latest'}}
-          run: pip install torch==${{ matrix.torch-version }}
-
+              pip install "numpy==1.26.4" scikit-learn flake8 setuptools numba
```

Comment:
curious, why are the other dependencies not pinned to a specific version?

Reply:
I've received an error from Torch - Numpy interaction and this is the version we have in OSS dependencies currently.

The public repository doesn't have any changes to the unit tests workflow for 2 years - they don't support Python 3.8 yet so I thought this is already much better than what there was 😛